hbw/config/defaults_and_groups.py

```diff
@@ -129,25 +129,34 @@ def set_config_defaults_and_groups(config_inst):
         "tt": ["tt_*"], "st": ["st_*"], "w": ["w_lnu_*"], "dy": ["dy_*"],
         "qcd": ["qcd_*"], "qcd_mu": ["qcd_mu*"], "qcd_ele": ["qcd_em*", "qcd_bctoe*"],
         "signal": ["ggHH_*", "qqHH_*"], "gghh": ["ggHH_*"], "qqhh": ["qqHH_*"],
-        "ml": [default_signal_dataset, "tt_*", "st_*", "dy_*", "w_lnu_*"],
+        "ml": ["ggHH_kl_1*", "tt_*", "st_*", "dy_*", "w_lnu_*"],
+        "dilep": ["tt_*", "st_*", "dy_*",  "w_lnu_*",
+            "ggHH_kl_0_kt_1_dl_hbbhww_powheg", "ggHH_kl_1_kt_1_dl_hbbhww_powheg",
+            "ggHH_kl_2p45_kt_1_dl_hbbhww_powheg", "ggHH_kl_5_kt_1_dl_hbbhww_powheg",
+        ],
     }
 
     # category groups for conveniently looping over certain categories
     # (used during plotting)
-    if signal_tag == "sl":
-        config_inst.x.category_groups = {
-            "much": ["1mu", "1mu__resolved", "1mu__boosted"],
-            "ech": ["1e", "1e__resolved", "1e__boosted"],
-            "default": ["incl", "1e", "1mu"],
-            "test": ["incl", "1e"],
-        }
+    config_inst.x.category_groups = {
+        "much": ["1mu", "1mu__resolved", "1mu__boosted"],
+        "ech": ["1e", "1e__resolved", "1e__boosted"],
+        "default": ["incl", "1e", "1mu"],
+        "test": ["incl", "1e"],
+        "dilep": ["incl", "ee", "2mu","emu"],
```

Comment:
For the sake of consistency, I think it would be good if the categories have similar naming. So either `2e`, `2mu` for the DL ones (to follow the convention of the SL ones) or `ee` and `mm` (or `mumu`). This change should be propagated everywhere these new categories are used.